components/Buttons/src/Theming/MDCButton+MaterialTheming.m

```diff
@@ -40,7 +40,19 @@ - (void)applyContainedThemeWithScheme:(nonnull id<MDCContainerScheming>)scheme {
 }
 
 - (void)applyContainedThemeWithColorScheme:(id<MDCColorScheming>)colorScheme {
-  [MDCContainedButtonColorThemer applySemanticColorScheme:colorScheme toButton:self];
+  [self resetButtonColorsForAllStates];
```

Comment:
The change to this file seems unrelated to deprecating MDCContainedButtonColorThemer. Please move this to a separate PR.

Reply:
It is related because the deprecated API was used here and now it is not.

Reply:
Done

Reply:
Please do not change Active libraries as part of this change – this PR should only be adding deprecation warnings. These changes should be a part a future PR to remove the class being deprecated in this PR.

Reply:
As discussed, future PRs will be separated.